raiden/tests/utils/tester_client.py

```diff
@@ -602,13 +602,15 @@ def deposit(self, amount):
         self.proxy.deposit(amount)
         self.tester_state.mine(number_of_blocks=1)
 
+    # opened() and closed() method return different values for geth and tester chain
```

Comment:
He miscommented. The new test he made was testing `client.opened()` which was giving a different response for tester and geth. His fix made it a consistent `''` between both.

Nothing was failing before because no test was testing a call of `opened()` after settling (which causes a selfdestruct)